main.py

```diff
@@ -50,11 +51,23 @@
 def http_request(request):
     now = dt.now()
     date_value = dt.strftime(now, "%Y-%m-%dT%H:%M:%S.%fZ")
+    message = ''
+    status_code = ''
     try:
         message, status_code =  main()
-    except logging.exception as err:
-        message, status_code =  "Fail", 410
-    message = (f'{message} on {date_value}.')
+    except Exception as err:
+        logging.exception(err)
+        exception_message =  tb.format_exc().splitlines()
+        message = exception_message[-1].capitalize()
+        message = message.replace('<',' ')
+        message = message.replace('>',' ')
```

Comment:
What do you think of adding something like this: 

```
message = "An error occurred. Details: " + message
```

Reply:
Change applied by commit [b598c82](https://github.com/kieras/zombie-projects-watcher/pull/63/commits/b598c82ed61e4da92ae116290b25fb3615d5586d)